packages/core/src/tsx/typeParam.ts

```diff
@@ -0,0 +1,15 @@
+import ts from "typescript"
+
+interface TypeParamProps {
+  name: string
+  // TODO constraint
```

Comment:
Should we do something with this now, or later on?

Reply:
When we need it :)